test/Dialect/FIRRTL/parse-errors.fir

```diff
@@ -9,6 +9,41 @@ circuit test :
 
 ;// -----
 
+FIRRTL
+; expected-error @+1 {{expected version after 'FIRRTL'}}
```

Comment:
The checks in this are running `expected-error` sometimes on the version string and sometimes on the circuit. What's going on here?

Reply:
Whether it's failing from parsing a version string or parsing "version".